promql/value.go

```diff
@@ -65,9 +65,10 @@ func (s Scalar) MarshalJSON() ([]byte, error) {
 
 // Series is a stream of data points belonging to a metric.
 type Series struct {
-	Metric     labels.Labels `json:"metric"`
-	Floats     []FPoint      `json:"values,omitempty"`
-	Histograms []HPoint      `json:"histograms,omitempty"`
+	Metric         labels.Labels `json:"metric"`
+	Floats         []FPoint      `json:"values,omitempty"`
+	Histograms     []HPoint      `json:"histograms,omitempty"`
+	ShouldDropName bool
```

Comment:
I'm wondering how this interacts with json marshalling (and if it should be marshaled at all in JSON). For now, let's add `` `json:"-"` `` as a tag to play safe.

In any case, this field could benefit from a comment what it is for.